src/main/java/frc/robot/subsystems/flywheel/FlywheelCommandBuilder.java

```diff
@@ -0,0 +1,46 @@
+package frc.robot.subsystems.flywheel;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.FunctionalCommand;
+import frc.utils.utilcommands.InitExecuteCommand;
+
+public class FlywheelCommandBuilder {
+
+	private Flywheel flywheel;
+
+	public FlywheelCommandBuilder(Flywheel flywheel) {
+		this.flywheel = flywheel;
+	}
+
+	//@formatter:off
+
+	public Command setPower(double power) {
+		return new FunctionalCommand(
+				() -> flywheel.setPower(power),
```

Comment:
move to execute

Reply:
why?

Reply:
not 100% sure, yoav told me to so i trust him, maybe ask him